doc/source/release-notes/Percona-Server-8.0.25-15.1.rst

```diff
@@ -0,0 +1,62 @@
+includes all the features and bug fixes available in the
+`MySQL 8.0.24 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-24.html>`__ and
+the
+`MySQL 8.0.25 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-25.html>`_
+in addition to enterprise-grade features developed by Percona.
+
+.. note::
+
+    The TokuDB Storage Engine was `declared as deprecated <https://www.percona.com/doc/percona-server/8.0/release-notes/Percona-Server-8.0.13-3.html>`__ in Percona Server for MySQL 8.0 and will be disabled in upcoming 8.0 versions. 
+    
+    We recommend migrating to the MyRocks Storage Engine. 
+
+    For more information, see the Percona blog post: `Heads-Up: TokuDB Support Changes and Future Removal from Percona Server for MySQL 8.0 <https://www.percona.com/blog/2021/05/21/tokudb-support-changes-and-future-removal-from-percona-server-for-mysql-8-0/>`__.
+
+
+New Features
+================================================================================
+
+* :jirabug:`PS-7182`: Create functionality to expose defined data from procfs for agentless environment
+* :jirabug:`PS-7671`: Add rocksdb_allow_unsafe_alter to enable crash unsafe INPLACE ADD|DROP partition
+* :jirabug:`PS-7327`: INPLACE ADD|DROP partitions in MyRocks
+
+
+
+Improvements
+================================================================================
+
+* :jirabug:`PS-7366`: Add jemalloc memory allocation profiling on PS 8.0
+
+
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PS-7722`: Multiple-Column Index using Column Prefix Key Parts fails with Index Condition Pushdown in MyRocks
+* :jirabug:`PS-7665`: The ``performance_schema.metadata_locks m_column_name_length`` is uninitialized for MDL_key::FOREIGN_KEY (Upstream :mysqlbug:`103532`)
+* :jirabug:`PS-7695`: The ``Boost`` download is no longer available. Using the -DDOWNLOAD_BOOST option with CMake (Thanks to user Benjamin Kuen for reporting this issue).
+* :jirabug:`PS-6802`: Configure fails with make-4.3 with CMake Error at storage/rocksdb/CMakeLists.txt:152 (STRING) (Thanks to user Thomas Deutschmann for reporting this issue).
+* :jirabug:`PS-7648`: Optimizer switch "favor_range_scan" is not documented
+* :jirabug:`PS-7595`: Same version upgrade from PS->PXC needs mysql_upgrade
+* :jirabug:`PS-7657`: A server exit caused with an update query on a partition table with a compressed column
+* :jirabug:`PS-7557`: Mysql server version 8.0.22-13 executing the Data Masking plugin causes a server exit (Thanks to user Alfonso Luciano for reporting this issue).
+* :jirabug:`PS-7526`: Unexpected quoting and dropping of comments in ``DROP TABLE`` commands (Upstream :mysqlbug:`87852`)
+* :jirabug:`PS-1116`: LP #1719506: Audit plugin reports "command_class=error" for server-side prepared statements
+* :jirabug:`PS-7789`: `procs_files_spec` can be edited with SET PERSIST_ONLY.
```

Comment:
This is probably not needed as it was not present in previous release and fixed in this one :)

Reply:
Removed.